test/Integration/IntegrationTestBase.t.sol

```diff
@@ -98,7 +98,12 @@ contract IntegrationTest is Test {
         vm.prank(owner);
         reverseRegistrar.setName("rootOwner");
 
-        vm.warp(GRACE_PERIOD * 10);
+        vm.prank(owner);
+        baseRegistrar.addController(address(registrarController));
+
+        vm.prank(owner);
+        registrarController.setLaunchTime(LAUNCH_TIME);
```

Comment:
@stevieraykatz what's the process for assuring `LAUNCH_TIME` is not set in the future relative to `block.timestamp`?

Also at launch maybe us a multicall approach or set the launch time on the `registrarController` before adding it to the `baseRegistrar` as a controller for any spam transactions trying to lodge them in between both (not sure if this is possible with the sequencer)

Reply:
We're unable to use multicall due to the ownership restrictions on both methods. We're going to set the launchTime to say 12:00PST and then execute the TX at 12:00 so we'll miss the first couple seconds of the auction but it won't be in the future.